<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation with Arrows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 图像显示 -->
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>770</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <!-- 上传图片按钮 -->
   <widget class="QPushButton" name="button_load_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <!-- 操作按钮 -->
   <widget class="QPushButton" name="button_draw_arrow">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Arrow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_undo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>120</width>
      <height>30</height>
    </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_screenshot">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>120</width>
      <height>30</height>
    </rect>
    </property>
    <property name="text">
     <string>Save Screenshot</string>
    </property>
   </widget>
   <!-- 日志窗口 -->
   <widget class="QTextEdit" name="log_window">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>770</width>
      <height>200</height>
    </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
